### PR TITLE
docs: docs for distributed cost center

### PR DESCRIPTION
**Pull request by KaviyaPeriyasamy**
- [X]  Documentation added for Distributed Cost Center

distributed_cost_center

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:1]: @KaviyaPeriyasamy add link in contents.md

File: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+metatags:
+ description: In ERPNext, a Distributed cost center enhances the cost center dimension to allow the distribution of the tagged value to a set of cost centers. A distributed cost center can be created out of different ratios of ordinary cost centers and it can be tagged against the GL entries.
+ keywords: frappe, erpnext, accounting reports, cost center, GL Entry.
+---
+
+<!-- add-breadcrumbs -->
+# Distributed Cost Center
+ 
+The Distributed cost center enhances the cost center dimension to allow the distribution of the tagged value to a set of cost centers. A distributed cost center can be created out of different ratios of ordinary cost centers and it can be tagged against the GL entries. Expense and income will not affect directly to the cost center. It will be distributed with the selected cost centers as 100% equal calculation.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:14]: add a brief definition in bold. What this is dist cost center exactly?

File: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## 1. How to create the Distributed Cost Center
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:16]:
```suggestion
## 1. How to create a Distributed Cost Center
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+The following reports will be automatically updated when cost center filter is added:
+
+  * [Accounting Reports](/docs/user/manual/en/accounts/accounting-reports)
+    * Financial Statements
+    * Budget Variance
+    * General Ledger
+  * [Profitability Analysis](/docs/user/manual/en/accounts/articles/tracking-project-profitability-using-cost-center)
+
+### 2. Related Topic
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:33]:
```suggestion
### 2. Related Topics
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+ keywords: frappe, erpnext, accounting reports, cost center, GL Entry.
+---
+
+<!-- add-breadcrumbs -->
+# Distributed Cost Center
+ 
+The Distributed cost center enhances the cost center dimension to allow the distribution of the tagged value to a set of cost centers. A distributed cost center can be created out of different ratios of ordinary cost centers and it can be tagged against the GL entries. Expense and income will not affect directly to the cost center. It will be distributed with the selected cost centers as 100% equal calculation.
+
+## 1. How to create the Distributed Cost Center
+1. Go to the cost center list, click on New.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:17]: Cost Center (C caps everywhere in text)

Comment (KaviyaPeriyasamy) [reply erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md]: @prasadarr Check my recent commits.

File: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+metatags:
+ description: A Distributed Cost Center is a Cost Center in which multiple Cost Centers are tagged with an appropriate percentage.
+ keywords: frappe, erpnext, accounting reports, Cost Center, GL Entry.
+---
+
+<!-- add-breadcrumbs -->
+# Distributed Cost Center
+
+**A Distributed Cost Center is a *Cost Center* in which multiple Cost Centers are tagged with an appropriate percentage.**
+
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:15]: @KaviyaPeriyasamy can you add an example, new readers will not understand what this, it's purpose

Comment (KaviyaPeriyasamy) [reply erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:15]: @prasadarr Hope this helps newbie https://github.com/frappe/erpnext_documentation/pull/85/commits/45ef27176197f8d22a69974bdaa8c0bd8e259c78

File: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+For example, you can mention the Cost Center 'A' as a Distributed Cost Center. when its income, expense, and budget are shared among various Cost Centers like 'B' and 'C' with some percentage equivalence.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:16]:
```suggestion
For example, you can mention the Cost Center 'A' as a Distributed Cost Center when its income, expense, and budget are shared among various Cost Centers like 'B' and 'C' with some percentage equivalence.
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+metatags:
+ description: A Distributed Cost Center is a Cost Center in which multiple Cost Centers are tagged with an appropriate percentage.
+ keywords: frappe, erpnext, accounting reports, Cost Center, GL Entry.
+---
+
+<!-- add-breadcrumbs -->
+# Distributed Cost Center
+
+**A Distributed Cost Center is a *Cost Center* in which multiple Cost Centers are tagged with an appropriate percentage.**
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:14]:
```suggestion
**A Distributed Cost Center is a Cost Center in which multiple Cost Centers are tagged with an appropriate percentage.**
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md (hunk 6 above)
Comment (prasadarr) [reply erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:15]: @KaviyaPeriyasamy it still doesn't tell me what's the benefit of having a distributed cost center. In how are the costs shared? equally? What problem does this feature solve?

Comment (KaviyaPeriyasamy) [reply erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:15]: @prasadarr Check whether https://github.com/frappe/erpnext_documentation/pull/85/commits/6c0d92b833b448089d96c9fc276a07e1b0f7bb13 it clarifies?

Comment (KaviyaPeriyasamy) [reply erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:15]: This is the exact thing solved by this feature.

File: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+For example, In your business, If the Cost Center 'B' and 'C' depend on Cost Center 'A' by 20% and 80%. Here, you can mention 'A' as a Distributed Cost Center. It helps to reflect income, expense, and budget of 'A' in 'B' and 'C' with allocated percentages.
+
+In ERPNext you can create Distributed Cost Center and use them in transactions and reports.
+
+## 1. How to create a Distributed Cost Center
+1. Go to the Cost Center list, click on New.
+1. Enter the Cost Center name.
+1. Select the parent Cost Center.
+1. Enable the checkbox, **Enable Distributed Cost Center**: On enabling this the distributed Cost Center table will show, Then select the Cost Center and allocate the corresponding percentage.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:24]:
```suggestion
1. Enable the checkbox, **Enable Distributed Cost Center**: On enabling this, the distributed Cost Center table will show. Here, select the Cost Centers and allocate the corresponding percentage.
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md (hunk 6 above)
Comment (KaviyaPeriyasamy) [reply erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:15]: Thanks!

Comment (prasadarr) [reply erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:15]: well, just one more thing. What business problem does this solve?
i.e., why would someone want to use this feature?

Comment (KaviyaPeriyasamy) [reply erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:15]: If a business has a master cost center with dependent cost centers. In every master cost center transactions, it is difficult to update the budget, profit, and loss to each dependent cost center manually with the allocated percentage of the master cost center. This feature helps to automate the process of manual entry.

Comment (prasadarr) [reply erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:15]: @KaviyaPeriyasamy 
please add this in the PR, I can't edit your file (tick "Allow edits from maintainers" in future PRs)

Comment (KaviyaPeriyasamy) [reply erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:15]: Sure. Kindly check this 60d1fcddc082fad73a53db916c1aa09bbe8efd0f

File: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+ description: A Distributed Cost Center is a Cost Center in which multiple Cost Centers are tagged with an appropriate percentage.
+ keywords: frappe, erpnext, accounting reports, Cost Center, GL Entry.
+---
+
+<!-- add-breadcrumbs -->
+# Distributed Cost Center
+
+**A Distributed Cost Center is a Cost Center in which multiple Cost Centers are tagged with an appropriate percentage.**
+
+If a business has a master Cost Center with dependent Cost Centers. In every master Cost Center transactions, it is difficult to update the budget, profit, and loss to each dependent Cost Center manually with the allocated percentage of the master Cost Center. This feature helps to automate the process of manual entry.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:16]:
```suggestion
If a business has a master Cost Center with dependent Cost Centers. In every master Cost Center transaction, it is difficult to update the budget, profit, and loss to each dependent Cost Center manually with the allocated percentage of the master Cost Center. This feature helps to automate the process of manual entry.
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+For example, In your business, If the Cost Center 'B' and 'C' depend on Cost Center 'A' by 20% and 80%. Here, you can mention 'A' as a Distributed Cost Center. It helps to reflect income, expense, and budget of 'A' in 'B' and 'C' with allocated percentages.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/distributed-cost-center.md:18]:
```suggestion
For example, In your business, If the Cost Center 'B' and 'C' depend on Cost Center 'A' by 20% and 80%. Then, you can mention 'A' as a Distributed Cost Center. It helps to reflect income, expense, and budget of 'A' in 'B' and 'C' with allocated percentages.
```

**Pull request merged**